site/src/site.js

```diff
@@ -1,15 +1,17 @@
-var React = require('react');
-var Router = require('react-router');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Router, Route, Link, IndexRoute } from'react-router';
+import { createHistory, useBasename } from 'history';
 
 const NavItems = [
-	{ value: 'css',         label: 'CSS' },
-	{ value: 'grid',        label: 'Grid' },
-	{ value: 'buttons',     label: 'Buttons' },
-	{ value: 'glyphs',      label: 'Glyphs' },
-	{ value: 'forms',       label: 'Forms' },
-	{ value: 'spinner',     label: 'Spinner' },
-	{ value: 'modal',       label: 'Modal' },
-	{ value: 'misc',        label: 'Misc' }
+	{ value: '/css',         label: 'CSS' },
+	{ value: '/grid',        label: 'Grid' },
+	{ value: '/buttons',     label: 'Buttons' },
+	{ value: '/glyphs',      label: 'Glyphs' },
+	{ value: '/forms',       label: 'Forms' },
+	{ value: '/spinner',     label: 'Spinner' },
+	{ value: '/modal',       label: 'Modal' },
+	{ value: '/misc',        label: 'Misc' }
```

Comment:
new react router gets rid of names, only way I could get stuff working was changing this value to an actual path.